### PR TITLE
dts: msm8916: mtp-smb1360: Add acer-a1-724

### PR DESCRIPTION
**Pull request by wonderfulShrineMaidenOfParadise**

### CODE REVIEW

File: dts/msm8916/msm8916-mtp-smb1360.dts [hunk 1]
```diff
@@ -9,6 +9,35 @@
...
+
+		smb1360 {
+			compatible = "qcom,smb1360";
+			i2c-gpio-pins = <14 15>;
+
+			qcom,batt-profile-select;
+			qcom,batt-id-vref-uv = <1800000>;
+			qcom,batt-id-rpullup-kohm = <100>;
+			qcom,profile-a-rid-kohm = <50>;
+			qcom,profile-b-rid-kohm = <200>;
```
Comment (wonderfulShrineMaidenOfParadise) [inline dts/msm8916/msm8916-mtp-smb1360.dts:37]: To be verified.

Comment (wonderfulShrineMaidenOfParadise) [reply dts/msm8916/msm8916-mtp-smb1360.dts]: Drop the battery selection for now.

File: dts/msm8916/msm8916-mtp-smb1360.dts [hunk 2]
```diff
@@ -9,6 +9,24 @@
 	qcom,msm-id = <206 0>;
 	qcom,board-id = <8 1>;
 
+	/*
+	* To get a working build for acer-a1-724, comment out
+	* $(LOCAL_DIR)/msm8916-mtp-smb1360.dtb in rules.mk in this directory.
```
Comment (stephan-gh) [inline dts/msm8916/msm8916-mtp-smb1360.dts:14]: Did you mean to comment out msm8916-mtp.dts?

Comment (wonderfulShrineMaidenOfParadise) [reply dts/msm8916/msm8916-mtp-smb1360.dts:14]: Not really. I mean "comment out all dtbs and add $(LOCAL_DIR)/msm8916-mtp-smb1360.dtb".

Comment (stephan-gh) [reply dts/msm8916/msm8916-mtp-smb1360.dts:14]: comment out **all lines except** ...

maybe then?

File: lk2nd/panel/generated/panels.h [hunk 3]
```diff
@@ -5,6 +5,7 @@
 
 #include "lk_panel_boe_otm8019a_5p0_fwvga_video.h"
 #include "lk_panel_booyi_otm1287_720p_video.h"
+#include "lk_panel_hx8394d_720p_video.h"
```
Comment (wonderfulShrineMaidenOfParadise) [inline lk2nd/panel/generated/panels.h:8]: Do I need to rename the panel name here as well?

Comment (stephan-gh) [reply lk2nd/panel/generated/panels.h:8]: No, it's fine here for now.

**Pull request merged**